app/controllers/api/base_controller.rb

```diff
@@ -5,6 +5,8 @@ module Api
   class BaseController < ApplicationController
     rescue_from ActiveRecord::RecordNotFound, with: :missing_record
 
+    private
```

Comment:
This change is unrelated to this pr, but I noticed it while working on the deployment. All public methods in controllers get turned into lambdas by Jets and there's no need for us to have a `missing_record` lambda function.